package.json

```diff
@@ -47,11 +47,13 @@
         "postinteg:local:nightwatch:chrome": "kill $(lsof -t -i:8080)",
         "preinteg:local:nightwatch:firefox": "http-server ./build/dev -s &",
         "integ:local:nightwatch:firefox": "nightwatch -e firefox",
-        "postinteg:local:nightwatch:firefox": "kill $(lsof -t -i:8080)"
+        "postinteg:local:nightwatch:firefox": "kill $(lsof -t -i:8080)",
+        "smoke": "npx playwright test"
```

Comment:
question: Under what situations will this target be executed? Specifically, I'm thinking about how we will execute these tests locally. Should we add an additional target which spins up a server locally, as we do for the other tests here?

Reply:
Yes, for local development purpose we do need that option; will add in the next revision.